docs/install/Knative-with-MicroK8s.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Install on MicroK8s"
+linkTitle: "MicroK8s"
+weight: 10
+type: "docs"
+---
+
+[MicroK8s](https://microk8s.io) is a lightweight, powerful fully-conformant Kubernetes that tracks upstream releases and makes clustering trivial. It can run on any flavor of Linux that supports [Snap](https://snapcraft.io) packages. It can run on Windows and Mac OS using [Mutlipass](https://multipass.run).
+This guide walks you through the installation of Knative using MicroK8s.
+
+You can find [guides for other platforms here](./README.md).
+
+### Install MicroK8s
+
+```shell
+   sudo snap install --classic microk8s
+```
+
+## Alias MicroK8s kubectl for convenience
+
+```shell
+   sudo snap alias microk8s.kubectl kubectl
+```
+
+## Enable Knative
+
+```shell
+   echo 'N;' | microk8s.enable knative
+```
+
+This command will install all available Knative v0.7.1 components.
```

Comment:
How do we prevent this documentation from falling out of date? The user seems to request the latest version and this document mentions the 0.7.1 version getting installed. 

Is there a way to query the version through the microk8s enablement interface or to select a particular version?

Reply:
Since there is only Knative v 0.7.1 that is available with MicroK8s, that's the only one available. When MicroK8s gets an updated version of Knative, a version info could be added.